src/emulator.cpp

```diff
@@ -204,8 +204,9 @@ void Emulator::LoadSystemModules(const std::filesystem::path& file) {
          {"libSceLibcInternal.sprx", &Libraries::LibcInternal::RegisterlibSceLibcInternal},
          {"libSceDiscMap.sprx", &Libraries::DiscMap::RegisterlibSceDiscMap},
          {"libSceRtc.sprx", &Libraries::Rtc::RegisterlibSceRtc},
-         {"libSceJpegEnc.sprx", nullptr}},
-    };
+         {"libSceJpegEnc.sprx", nullptr},
+         {"libSceRazorCpu.sprx", nullptr},
```

Comment:
This lib unstable no? Should be in `src/core/libraries/libs.cpp` instead to prevent calls to nonexistent hardware if someone use this LLE.

Reply:
libSceRazorCpu LLE should be fine, I haven't found any potential issues when looking at it in Ghidra, or when using it in games.